### PR TITLE
ACMS-1655: Unable to run blt setup. Config rewrite issue.

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #[ACMS-1655](https://backlog.acquia.com/browse/ACMS-1655)

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by deepakmishra2 — APPROVED** [review]
Looks good to me

File: modules/acquia_cms_site_studio/acquia_cms_site_studio.install [hunk 1]
```diff
@@ -120,6 +122,24 @@ function acquia_cms_site_studio_install($is_syncing) {
...
     $config_storage->write('editor.editor.cohesion', $source_optional_dir->read('editor.editor.cohesion'));
     $config_storage->write('filter.format.cohesion', $source_optional_dir->read('filter.format.cohesion'));
+
+    // Re-write filter format filtered html.
+    $editorObject = $entity_type_manager
+      ->getStorage('editor')
+      ->load('filtered_html');
+    if ($editorObject instanceof Editor) {
+      $filtersObj = $editorObject->getFilterFormat();
+      $filters_instance_ids = $editorObject->getFilterFormat()->filters()->getInstanceIds();
```
Comment (apathak18) [inline modules/acquia_cms_site_studio/acquia_cms_site_studio.install:132]: @chandan-singh7929 replace **$editorObject->getFilterFormat()** with $filtersObj.
As discussed we have to follow same pattern throughout the codebase. 
.install .module file variables naming convention are in snake_case
classes plugins services should be in camelCase. Please make changes accordingly.

Comment (chandan-singh7929) [reply modules/acquia_cms_site_studio/acquia_cms_site_studio.install]: @apathak18 - I have made the changes, thank you!

**Review by rajeshreeputra — APPROVED** [review]

File: modules/acquia_cms_site_studio/tests/src/Functional/FilterFormatFilteredHtmlTest.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    $assert_session = $this->assertSession();
+    $account = $this->drupalCreateUser();
+    $account->addRole('administrator');
+    $account->save();
+    $this->drupalLogin($account);
+
+    // Visit the filter page.
+    $this->drupalGet('/admin/config/content/formats/manage/filtered_html');
+    $assert_session->statusCodeEquals(200);
+    $assert_session->elementExists('css', '#edit-filters-black-list-html-tags-status');
```
Comment (vishalkhode1) [inline modules/acquia_cms_site_studio/tests/src/Functional/FilterFormatFilteredHtmlTest.php:55]: @chandan-singh7929 Also verify that element is unchecked (as default status is FALSE).

**Review by vishalkhode1 — CHANGES_REQUESTED** [review]

File: modules/acquia_cms_site_studio/tests/src/Functional/FilterFormatFilteredHtmlTest.php (hunk 2 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_site_studio/tests/src/Functional/FilterFormatFilteredHtmlTest.php]: Done

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**